app/main/plugins/core/basic-apps/linux.js

```diff
@@ -0,0 +1,125 @@
+import { remote } from 'electron'
+import path from 'path'
+import fs from 'fs'
+import uniq from 'lodash/uniq'
+import flatten from 'lodash/flatten'
+import { shellCommand } from 'cerebro-tools'
+
+let appDirs = [
+  path.join(remote.app.getPath('home'), '.local', 'share'),
+  path.join('/usr', 'share'),
+  path.join('/usr', 'share', 'ubuntu'),
+  path.join('/usr', 'share', 'gnome'),
+  path.join('/usr', 'local', 'share'),
+  path.join('/var', 'lib', 'snapd', 'desktop')
+]
+
+if (!!process.env.XDG_DATA_DIRS) {
+  appDirs = [
+    ...appDirs,
+    ...process.env.XDG_DATA_DIRS.split(':')
+  ]
+}
+
+// Icon resolutions in priority of checking
+const iconResolutions = [
+  '1024x1024',
+  '512x512',
+  '256x256',
+  '192x192',
+  '128x128',
+  '96x96',
+  '72x72',
+  '64x64',
+  '48x48',
+  '40x40',
+  '36x36',
+  '32x32',
+  '24x24',
+  '22x22',
+  '20x20',
+  '16x16'
+]
+
+// Directories when we are trying to find an icon
+const iconDirs = uniq(flatten([
+  ...iconResolutions.map(resolution => (
+    appDirs.map(dir => path.join(dir, 'icons', 'hicolor', resolution, 'apps'))
+  )),
+  path.join('/usr', 'share', 'pixmaps'),
+  path.join('/usr', 'share', 'app-install', 'icons')
+])).filter(fs.existsSync)
+
+const svgIconDirs = [
+  path.join('/usr', 'share', 'icons', 'hicolor', 'scalable', 'apps')
+].filter(fs.existsSync)
```

Comment:
@KELiON I think you should consider reusing `iconDirs`, there you have a good set of many possible icon locations, I'm sure they cover more possibilities. I'm simply not sure that `/usr/share/icons/hicolor/scalable` is the only possible location.

Perhaps just add `scalable` as the first resolution, then `iconDirs` would visit these folders as well.